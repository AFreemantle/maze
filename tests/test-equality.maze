class test {
    void main (string args) {
        
	if(42<50){
            print("less");
        }

        if(60>3){
	    print("greater");
  	}

	if(100==100){
	    print("equal");
	}

	if(30!=53){
	    print("not equal");
	}

	if(10<=40){
	    print("leq");
	}

	if(50>=50){
	    print("geq");
	}
	
    }
}
