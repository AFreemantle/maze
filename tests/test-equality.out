less
greater
equal
not equal
leq
geq
